src/main/java/io/github/syst3ms/skriptparser/util/math/BigRational.java

```diff
@@ -214,8 +214,7 @@ public BigRational multiply(BigRational value) {
 	// private, because we want to hide that we use BigDecimal internally
 	private BigRational multiply(BigDecimal value) {
 		BigDecimal n = numerator.multiply(value);
-		BigDecimal d = denominator;
-		return of(n, d);
+		return of(n, denominator);
```

Comment:
Why only go halfway? There's no problem with inlining the other term.

Reply:
Sure.